### PR TITLE
Refs codership/mysql-features#33

### PR DESCRIPTION
**Pull request by ayurchen**
- this fixes a bug where wsrep_start_position_update() would pass local_uuid and local_seqno to wsrep_sst_received() instead of real, submitted position.

### CODE REVIEW

File: sql/wsrep_var.cc [hunk 1]
```diff
@@ -147,26 +147,30 @@ bool wsrep_start_position_check (sys_var *self, THD* thd, set_var* var)
...
+static
+void wsrep_set_local_position(const char* const value, bool const sst)
 {
-  size_t value_len  = strlen (value);
-  size_t uuid_len   = wsrep_uuid_scan (value, value_len, &local_uuid);
-  local_seqno       = strtoll (value + uuid_len + 1, NULL, 10);
+  size_t const value_len(strlen(value));
+  wsrep_uuid_t uuid;
+  size_t const uuid_len(wsrep_uuid_scan(value, value_len, &uuid));
+  wsrep_seqno_t const seqno(strtoll(value + uuid_len + 1, NULL, 10));
```
Comment (temeo) [inline sql/wsrep_var.cc:157]: Variable initialization is not MySQL style.

**Comment by temeo** [comment]
Looks OK besides couple of style issues.

File: sql/wsrep_var.cc (hunk 1 above)
Comment (ayurchen) [inline sql/wsrep_var.cc:157]: MySQL style is not initialization but ASSIGNMENT! ;) But will fix that anyway.

**Comment by ayurchen** [comment]
Philip, reassigning to you as was discussed in chat. Teemu's concerns fixed.

**Pull request merged**